packages/server/lib/modes/run.ts

```diff
@@ -55,8 +56,8 @@ const relativeSpecPattern = (projectRoot, pattern) => {
   return pattern.map((x) => x.replace(`${projectRoot}/`, ''))
 }
 
-const iterateThroughSpecs = function (options: { specs: SpecFile[], runEachSpec: RunEachSpec, beforeSpecRun?: BeforeSpecRun, afterSpecRun?: AfterSpecRun, config: Cfg }) {
-  const { specs, runEachSpec, beforeSpecRun, afterSpecRun, config } = options
+const iterateThroughSpecs = function (options: { specs: SpecFile[], runEachSpec: RunEachSpec, beforeSpecRun?: BeforeSpecRun, afterSpecRun?: AfterSpecRun, config: Cfg, protocolManager?: any }) {
```

Comment:
Add type to `protocolManager`

Reply:
[`fcb6c6f` (#26080)](https://github.com/cypress-io/cypress/pull/26080/commits/fcb6c6f9833d27a6958eb0a8dbde8449fabc6447#diff-fae7886d3dd070e07a8b39ac5718534511764b7f03bc24c9726f82977b53a225)